src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -212,10 +212,9 @@ void BaseQNAcceleration::updateDifferenceMatrices(
       }
 
       PRECICE_CHECK(not math::equals(residualMagnitude, 0.0),
-                    "Attempting to add a zero vector to the quasi-Newton V matrix. This means that the residual "
-                    "in two consecutive iterations is identical. There is probably something wrong in your adapter. "
-                    "Maybe you always write the same (or only incremented) data or you call advance without "
-                    "providing  new data first.");
+                    "Attempting to add a zero vector to the quasi-Newton V matrix. This means that the residuals "
+                    "in two consecutive iterations are identical. If a relative convergence limit was selected, "
+                    "consider increasing the convergence threshold.");
```

Comment:
Why removing the previous hints? They are still helpful, no?
On the other hand, I would not mention the relative convergence limit here. We already have the warning below. Could only confuse here more than it helps?
If a user runs into this error message, I guess it is still most of the times due to

> There is probably something wrong in your adapter. Maybe you always write the same (or only incremented) data or you call advance without providing  new data first.

Reply:
If you write the same data, the first error should be seen:
https://github.com/precice/precice/blob/d1800d5801256861c7672907020a11d9bf4a2cbd/src/acceleration/BaseQNAcceleration.cpp#L178-L183

But if the change in residual is becoming very low, then you are nearing convergence.

Reply:
> If you write the same data, the first error should be seen

True.

Reply:
Should we leave this part as is now?

Reply:
> On the other hand, I would not mention the relative convergence limit here. We already have the warning below. Could only confuse here more than it helps?

This concern still holds. Do you agree?

Reply:
Well above, you only get a warning

> The coupling residual equals almost zero ...

One would still run into this error.
But in the end, not soo important.